### PR TITLE
added touch support

### LINKED ISSUE #16: feat: support for touch moves
Currently the knob does support mouse drags, mouse wheel events and up/down keys. On touch screens however, the value can not currently be changed. This feature should bring touch support similar to mouse drags.

@Uninen: I have already a solution for this in my fork's main repo. You might take it from there or, if you prefer, I can create a PR in a separate feature branch.

### PR DESCRIPTION
**Pull request by suterma**
Just a it says, touch now works similar to mouse drag.

Fixes #16

### CODE REVIEW

**Comment by Uninen** [comment]
Thanks for this! 👍 

I'll try to find some time soon to test this out properly before before I merge it.

File: src/ControlKnob.vue [hunk 1]
```diff
@@ -129,17 +129,28 @@ const mouseMoved = ref(false)
...
+  preventScrolling(event);
+}
+
+ /** Gets the y coordinate associated with the event */
+function getEventY(event: TouchEvent | MouseEvent): number {
+  if (window.TouchEvent && event instanceof TouchEvent) {
+    return event.touches[0].pageY;
+  }
+  else if (event instanceof MouseEvent) {
+    return currentY = event.clientY
```
Comment (Uninen) [inline src/ControlKnob.vue:145]: Return shouldn't contain assignment.

File: src/ControlKnob.vue [hunk 2]
```diff
@@ -210,6 +221,7 @@ function changeValue(change: number) {
 }
 
 function keyDownListener(event: KeyboardEvent) {
+  //Update the shift modifier here already, otherwise the precise mode is not triggered properly
```
Comment (Uninen) [inline src/ControlKnob.vue:224]: This is a good comment to have but is formatted incorrectly (missing a space at the end). Eslint should have catched this -- are you sure your editor is configured properly?

**Review by Uninen — CHANGES_REQUESTED** [review]
I appreciate the contribution, thank you 👍 

Happy to merge after these linting issues are resolved.

File: src/ControlKnob.vue (hunk 1 above)
Comment (suterma) [reply src/ControlKnob.vue:145]: I will add parenthesis around the assignment to make the intention clear. If you really want to have a separate line, this is fine with me, too. Just update the PR. You should have write access anyway.

File: src/ControlKnob.vue (hunk 2 above)
Comment (suterma) [reply src/ControlKnob.vue:224]: Done.

File: src/ControlKnob.vue (hunk 1 above)
Comment (Uninen) [reply src/ControlKnob.vue]: I'd rather separate the assignment to another line. It's much more clear and explicit. I'm not pointing this out just to be nitpicky, it's just general good practice and makes the code more readable. See: https://eslint.org/docs/latest/rules/no-return-assign

The messiness of JS is one of my personal pet peeves about it as it usually leads to bugs, and in a FOSS project like this readability matters even more :)

**Comment by suterma** [comment]
The assignment is now separate, to make the code more readable.

**Review by Uninen — APPROVED** [review]

**Pull request merged**